### ISSUE TITLE
Fix snapping for minimised controllers

### ISSUE THREAD

**Issue by chrisgoringe** [chrisgoringe/cg-controller#337]
Not certain what this means....

**Comment by JorgeR81** [comment]
Do you mean moving controllers together when they are collapsed?
This may not be necessary.

---

The current snapping for minimised controllers, it's working well, for me I already can:
- Snap a collapsed controller to the Top bar
- Snap a collapsed controller to other Controllers

This is very useful to change the position of snapped controllers. I do this:
- collapse a Controller,
- detach it
- replace with another collapsed Controller ( still will snap in place )
- expand the new Controller
- adjust the Controller's size ( if necessary ), to snap it to the the other Controllers

---


![row1](https://github.com/user-attachments/assets/f331c2dc-f22a-46b9-b063-71931b0e1462)


----

![s1](https://github.com/user-attachments/assets/3cab59a6-7594-4a89-ac72-120862600bdc)

![s06](https://github.com/user-attachments/assets/b62b4fd9-9fc3-4f1f-8301-20eb4ac311ca)

![s05](https://github.com/user-attachments/assets/62f9afb2-4c04-471a-858b-f3f03b688845)

![s3](https://github.com/user-attachments/assets/1969b7fb-e6d9-4a72-8a44-c9791b4b8dd9)

![s4](https://github.com/user-attachments/assets/bfda3f93-7009-43ec-956e-5a1d32524d25)

**Comment by chrisgoringe** [comment]
At present a collapsed controller snaps as if it was still full size.

I think that is correct behaviour, except that it probably ought to remain full width when collapsed (ie collapse should be vertical only, like it is for nodes with the controller)

**Comment by chrisgoringe** [comment]
Or maybe it's ok as it is!

**Comment by chrisgoringe** [comment]
Agree. This is closed 😀

**Comment by JorgeR81** [comment]
Changed my mind !

While building the reddit workflow, I realized snapping minimized windows is very useful.

Here I have 2 stacked windows in the same column.
I want the minimized "Conditioning" window, to be moved to the top, when I minimize the window above. 

I've just moved it myself for the Reddit images, and hoped nobody will notice :)

---

![z3](https://github.com/user-attachments/assets/c54c746a-ff24-47b6-9e1f-2a803cfd1dcc)

![z1](https://github.com/user-attachments/assets/7e1bd73a-54bf-42ae-8899-1ed84e921930)

![z0](https://github.com/user-attachments/assets/d9524a19-58e7-4b13-92ab-48345c33ee62)
